roles/translator/src/lib/downstream_sv1/diff_management.rs

```diff
@@ -244,21 +243,33 @@ impl Downstream {
                     || (hashrate_delta_percentage >= 30.0) && (delta_time >= 240)
                     || (hashrate_delta_percentage >= 15.0) && (delta_time >= 300)
                 {
-                if realized_share_per_min < 0.01 {
+                if realized_share_per_min == 0.0 {
```

Comment:
this is a float are you sure that is safe to change from < to == 

If yes write a comment that explain why